objects/rct2ww/scenery_group/rct2.ww.scgafric.json

```diff
@@ -127,7 +127,8 @@
             "cs-CZ": "Afrika",
             "ko-KR": "아프리카 테마",
             "pl-PL": "Wystrój afrykański",
-            "ru-RU": "Тема Африки"
```

Comment:
You spelt "africana" with a lower case here, but you spelt all the other continents with a capital letter.